app/forms/SupportingInformationDetailsFormProvider.scala

```diff
@@ -16,17 +16,15 @@
 
 package forms
 
-import javax.inject.Inject
-
 import forms.mappings.Mappings
+import javax.inject.Inject
 import play.api.data.Form
 
 class SupportingInformationDetailsFormProvider @Inject() extends Mappings {
 
   def apply(): Form[String] =
     Form(
       "value" -> text("supportingInformationDetails.error.required")
-        .verifying(maxLength(100, "supportingInformationDetails.error.length"))
```

Comment:
whats this change for?

Reply:
Ah - the 2nd part of your PR title I guess

Reply:
Aimee requested it the other day, I forgot to do a PR for it

Reply:
Bit of feature creep I apologise :)